apps/explorer/lib/explorer/chain/block.ex

```diff
@@ -24,6 +24,18 @@ defmodule Explorer.Chain.Block.Schema do
                               field(:parent_delta_s, :binary)
                               field(:parent_entropy_full, {:array, :binary})
                               field(:parent_delta_s_full, {:array, :binary})
+                              field(:evm_root, Hash.Full)
+                              field(:utxo_root, Hash.Full)
+                              field(:etx_set_hash, Hash.Full)
+                              field(:parent_uncled_sub_delta_s, {:array, :binary})
+                              field(:efficiency_score, :binary)
+                              field(:threshold_count, :binary)
+                              field(:expansion_number, :binary)
+                              field(:etx_eligible_slices, Hash.Full)
+                              field(:prime_terminus, Hash.Full)
+                              field(:interlink_root_hash, Hash.Full)
+                              field(:uncled_s, :binary)
+                              field(:interlink_hashes, {:array, Hash.Full})
```

Comment:
Not sure, but maybe will good idea to keep all work object in db and add label with name of parent field near value, for example label [WorkObject] 0x2123

Reply:
Could you elaborate a bit more on this suggestion? I didn't get what is the improvement here.